### PR TITLE
[391] AcceptAction should be available inside an Action body

### PR DESCRIPTION
**Pull request by jerome-obeo**
Bug: https://github.com/eclipse-syson/syson/issues/391

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -61,6 +61,7 @@ A reset of the database is needed.
 - https://github.com/eclipse-syson/syson/issues/371[#371] [diagrams] Handle the ForkNode concept in actions body of diagrams
 - https://github.com/eclipse-syson/syson/issues/381[#381] [diagrams] Handle the MergeNode concept in actions body of diagrams
 - https://github.com/eclipse-syson/syson/issues/389[#389] [diagrams] Handle the DecisionNode concept in actions body of diagrams
+- https://github.com/eclipse-syson/syson/issues/391[#391] [diagrams] AcceptAction should be available inside an Action body
```
Comment (AxelRICHARD) [inline CHANGELOG.adoc:64]: If it is a new feature, then you should not use "should" in your statement. "should" is rather for bug fixes.

Comment (AxelRICHARD) [reply CHANGELOG.adoc:64]: Or if it is a bug fix, please move it to the appropriate section, and update the message with something like : `Fix an issue where the AcceptAction were not available inside an Action body`

Comment (jerome-obeo) [reply CHANGELOG.adoc]: Agree. I've changed the issue headline to `AcceptAction is now available inside an Action body`.

Comment (jerome-obeo) [reply CHANGELOG.adoc]: Actually, this is not an issue since it is part of the action flow compartment development.

Comment (AxelRICHARD) [reply CHANGELOG.adoc]: Ok

**Review by AxelRICHARD — APPROVED** [review]

**Pull request merged**